app/src/controllers/plugins/sagas.js

```diff
@@ -114,6 +120,15 @@ function* updateIntegration({ payload: { data, isGlobal, pluginName, id, callbac
       ...data,
       integrationParameters: omit(data.integrationParameters, meta[SECRET_FIELDS_KEY]),
     };
+
+    const integrationParametersPath = integration.integrationParameters;
+    if (pluginName === SAML && integrationParametersPath[FULL_NAME_ATTRIBUTE_KEY]) {
```

Comment:
In this file we have a common logic for any type of integration. Do we really need to specify `SAML` related checks right here? Can it be done on the integration configuration level?